src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -586,80 +608,106 @@ public static boolean shouldFlip(Unit u)
   private void drawHUD(Graphics g, boolean includeTileDetails)
   {
     // Choose the CO overlay location based on the cursor location on the screen.
-    if( !overlayIsLeft && (myGame.getCursorX() - mapViewDrawX.get()) > (mapTilesToDrawX - 1) * 3 / 5 )
+    final double viewRelativeCursorX = myGame.getCursorX() - mapViewDrawX.get();
+    if( !overlayIsLeft && viewRelativeCursorX > (mapTilesToDrawX - 1) * 3 / 5 )
     {
       overlayIsLeft = true;
     }
-    if( overlayIsLeft && (myGame.getCursorX() - mapViewDrawX.get()) < mapTilesToDrawX * 2 / 5 )
+    if( overlayIsLeft && viewRelativeCursorX < mapTilesToDrawX * 2 / 5 )
     {
       overlayIsLeft = false;
     }
 
     // If the CO overlay won't overlap the map, draw all CO overlays to use the space
-    int overlayHSpaceAvailable = (mapViewWidth - mapImage.getWidth()) / 2;
-    if( overlayHSpaceAvailable > SpriteLibrary.getCoOverlay(myGame.activeArmy.cos[0], true).getWidth() + 42 )
+    final int oppositeHudSize = getOppositeHudSize();
+    final int overlayHSpaceAvailable = mapViewWidth - mapImage.getWidth() - oppositeHudSize;
+    final boolean drawAllHeaders = overlayHSpaceAvailable > COMPREHENSIVE_HUD_H_SIZE;
+
+    boolean drawOppositeHud = mapViewWidth - mapImage.getWidth() > 2 * oppositeHudSize;
+    // If we're in a menu and the cursor isn't too close to the edge, draw the opposite HUD even if it'll overlap the map
+    if( !drawOppositeHud && null != mapController.getCurrentGameMenu() )
     {
-      // We have plenty of space to draw everything, so default to drawing left
+      final int showInMenuThreshold = 4;
+      if( !overlayIsLeft && viewRelativeCursorX > showInMenuThreshold )
+      {
+        drawOppositeHud = true;
+      }
+      if( overlayIsLeft && mapTilesToDrawX - viewRelativeCursorX > showInMenuThreshold )
+      {
+        drawOppositeHud = true;
+      }
+    }
+
+    if( drawAllHeaders )
       overlayIsLeft = true;
 
+    if( drawOppositeHud )
+    {
+      drawTurnCounter(g, overlayIsLeft);
+      drawOppositeDetails(g, overlayIsLeft);
```

Comment:
So now the coords will only be drawn if the window is expanded, even if the user toggles coords on in options?
If the user wants coords, the user should get coords.

I don't think the coords or the turn counter are so in the way that we have to ban them. Even if they completely covered the corner squares (which they don't), those squares are never (I guess maybe "almost never", technically?) game-relevant anyway.
Your comment about not being able to see what's under them also isn't even true, since we have the tile overlay.

I really don't understand why you are so hostile to having them there, to the point where you are overriding user preferences to banish them.

Reply:
They're also drawn any time a menu is up. I guess we could change the option to "always show opposite UI/details/something"?
I've found them to get in the way at default size, and I quite like playing with the map/window perfectly sized to each other. It is map dependent whether it's a problem, though.

They're not banned. Also, they do cover menu options and damage popups sometimes, which is a real drag since those are smaller and don't show up on the details pane.
As for the details pane working to show you what's there... I guess that's true. It's pretty bad QoL, though.

A user complained about it, and I realized I really didn't like it either.

Reply:
User complaints are more valid than arbitrary preference changes. That said, there is an option to hide it.

> they do cover menu options and damage popups sometimes

Huh. I would have thought the menus were already being drawn last already. That would be annoying.

I'd probably be fine with grouping them in an "always show details" option or whatever, but then I'm sure someone would want to hide one but not the other. Guess we could wait for the complaint though. I guess another option is to just not have the option, and don't show that stuff unless there is room/when menus are up. Neither is exactly crucial information.

Reply:
There's no option to hide the turn counter, and of the two that's the guiltier party (damage popups for the top *two* rows hide under it)

Nah, menus are drawn on the map, from back when we didn't draw anything outside the map.
There wasn't a pressing need to change that, so...

Okay. Do you have a preference between "no option" and "always show option"? If you don't, I'll probably do whichever's easier.

Reply:
No strong preference.